### PR TITLE
Made changes to README.md

### PR DESCRIPTION
**Pull request by anishaJaiswal**
Trying out pull request.

### CODE REVIEW

**Review by siddheshphatak — CHANGES_REQUESTED** [review]
Checking the request change option

**Review by anishaJaiswal — COMMENTED** [review]
What changes ?

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
-# temp
+# temp 
+trying out pull requests
```
Comment (siddheshphatak) [inline README.md:2]: This is where you need to make that change

**Review by siddheshphatak — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (anishaJaiswal) [reply README.md:2]: OKey

Comment (anishaJaiswal) [reply README.md:2]: Made the requested changes

**Review by siddheshphatak — APPROVED** [review]
All changes looks fine 👍

**Pull request merged**